gnd/src/main/java/com/google/android/gnd/ui/home/featuresheet/ObservationListViewModel.java

```diff
@@ -59,14 +59,22 @@ public ObservationListViewModel(ObservationRepository observationRepository) {
   }
 
   /** Loads a list of observations associated with a given feature. */
-  public void loadObservationList(Feature feature, Form form) {
+  public void loadObservationList(Feature feature) {
+    ImmutableList<Form> forms = feature.getLayer().getForms();
     loadObservations(
-        feature.getProject(), feature.getLayer().getId(), form.getId(), feature.getId());
+        feature.getProject(),
+        forms.isEmpty() ? Optional.empty() : Optional.of(forms.get(0).getId()),
```

Comment:
Are we eventually going to support multiple forms again, or will `forms` eventually become `form`?

Reply:
The idea is that we'll keep it in the data store schema, but not in the UIs. This way custom or future clients can use multiple forms without breaking the database.

Reply:
Gotcha, maybe this is worthy of a helper method to make sure we always call `.get(0)` for now to fetch only the first form and don't accidentally call `.get(0±n)`? (Not that I anticipate us needing to call this in many other places in the app atm).

Reply:
I'll go you one better - pushing the cardinality change to the persistence layers, so that the application model doesn't reflect this idiosyncrasy in the data models. Please see latest commit 56cfa3df66cd7312961db3633c0fd06e919750c1.